testsuite/Makefile

```
@@ -15,7 +15,11 @@
 
 .NOTPARALLEL:
 
-BASEDIR := $(shell pwd)
+BASEDIR := "$(shell pwd)"
+ifneq "$(shell echo $(BASEDIR) | grep ' ')" ""
```

This command shells out two processes each time the `Makefile` is invoked, which can be a bit expensive. Instead, you can use pure `make`:
```suggestion
ifneq "$(words $(BASEDIR))" "1"
```

---

Thanks for the change I'm quite new with makefiles.